lib/open_weather_client.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'rest-client'
+
+class OpenWeatherClient
+  def initialize
+    @api_key = ENV['API_KEY']
+    @api_url = ENV['API_URL']
+  end
+
+  def check_weather(city_id)
```

Comment:
the consumer of the client doesn't necessarily know about what id a specific city has on the Open Weather API.
That knowledge belongs to the client, it is the one that knows that API-related kind of thing. 
The client encapsulates the knowledge about the API.

So I'd pass the city here instead of the id, and let the client convert that to whatever id that city is represented with in the API. 

You could maybe store that conversion city => id in a hash.
Or if you want to cover many cities, you may have it in a `.yml` configuration file, and handle it with the YAML native Ruby library.

If you don't want to add a lot of cities at this point, you can just have London for now.